CHANGELOG.md

```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v13.4.0](https://github.com/voxpupuli/puppet-rabbitmq/tree/v13.4.0) (2024-05-19)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-rabbitmq/compare/v13.3.0...v13.4.0)
+
+Historically we used the garethr/erlang module as soft dependency on CentOS 7. This also configured the EPEL7 repository. I always always a bit unsure where the actual rabbitmq packages come from. I assumed somehow from the erlang repo. My impression is that the erlan repo isn't used at all. rabbitmq packages where always pulled from EPEL7. The problem with the garethr/erlang is that it depends on the deprecated stahnma/epel (which is now puppet/epel) and uses topscope facts and
+variables. The module doesn't work anymore with Puppet 8.
```

Comment:
Do we want this whole note here?

Reply:
I thought it might confuse people when there's no note, but I don't have a strong opinion. If you want to can reword it at https://github.com/voxpupuli/puppet-rabbitmq/issues/984

Reply:
What about this wording:
> Historically we used the garethr/erlang module as soft dependency on CentOS 7. This also configured the EPEL7 repository. We have replaced this with an include of the EPEL repo on CentOS 7.

(I updated in #984, but feel free to revert to your wording or add stuff if you think it's missing something)

Reply:
I think the bit about puppet 8 not working is now not the case because of #985?

Reply:
I approved the PR, though if you use this version, you should rebuild the changelog - otherwise, maybe revert it back so it won't cause issues later?